tests/lac/solver_control_06.expect=run.debug.output

```diff
@@ -0,0 +1,2 @@
+
+content irrelevant
```

Comment:
Oh, I missed that. In fact, I did not even know that that can be done. Thanks for pointing it out!